Add file attributes for Genera

Symbolics opened the pull request:
Genera doesn't handle (in-package ...) the way that most people expect, but instead uses the Package file attribute to determine which package to place definitions. This is easily fixed by adding a comment line of the form:
```lisp
;;;; -*- Mode: LISP; Base: 10; Syntax: ANSI-Common-lisp; Package: CL-USER -*-
```
at the top of the file, with no change to the lisp code required.

phoe commented on tests.lisp:
```diff
@@ -1,7 +1,4 @@
-;;;; -*- Mode: Lisp; indent-tabs-mode: nil -*-
-
-(defpackage :split-sequence/tests
```
Since you remove the `DEFPACKAGE SPLIT-SEQUENCE/TESTS` form altogether, are you sure that this file compiles properly? If you tell Genera to use `Package: SPLIT-SEQUENCE/TESTS` then where is this package defined?

Symbolics commented:
That looks like an error. The package definition should be in a separate file. I'm away from my lisp machine for the holidays, so best we put this one on hold until I get back. Since it is a test, the test-package.lisp is a good convention for package definitions related to testing; I suggest we put it there.

    On Sunday, December 15, 2019, 10:22:19 AM EST, phoe <notifications@github.com> wrote:  
 
 
@phoe commented on this pull request.

In tests.lisp:
> @@ -1,7 +1,4 @@
-;;;; -*- Mode: Lisp; indent-tabs-mode: nil -*-
-
-(defpackage :split-sequence/tests

Since you remove the DEFPACKAGE SPLIT-SEQUENCE/TESTS form altogether, are you sure that this file compiles properly? If you tell Genera to use Package: SPLIT-SEQUENCE/TESTS then where is this package defined?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.

sionescu commented:
Ping.

Symbolics commented:
I'm not using Genera in my current work. Let's close this one and when I get back to Genera I'll resubmit.

    On Friday, January 1, 2021, 11:12:38 AM GMT+8, Stelian Ionescu <notifications@github.com> wrote:  
 
 


Ping.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
